tinygrad/codegen/uops.py

```diff
@@ -394,12 +644,30 @@ def _dfs(u:UOp, gate:UOp) -> UOp:
       if (replace_source:=tuple(_dfs(x, gate) for x in u.src)) != u.src: return UOp(u.op, u.dtype, replace_source, u.arg)
       return u
     for i, s in enumerate(self.sinks[:]):
-      if s.op is UOps.STORE and len(s.src) == 4 and (rw:=_dfs(s, s.src[3])) != s: self.sinks[i] = UOp(rw.op, rw.dtype, rw.src[:3], rw.arg)
+      # breaks for TC. maybe move WMMA to the lowerer?
+      if all(x.op is not UOps.TC for x in s.parents):
```

Comment:
I might be missing something, but is this still needed?

Reply:
ohhh, that's actually a bug now I think, but it's WMMA, not TC!